Lesson_2_Python_II/Lesson_2_clean.ipynb

```diff
@@ -6,49 +6,34 @@
    "source": [
```

Comment:
<p>We can modify the ticks on the x and/or y axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>